examples/uefi/libvirt.tf

```diff
@@ -0,0 +1,43 @@
+provider "libvirt" {
+    uri = "qemu:///system"
+}
+
+resource "libvirt_network" "dev-cluster" {
+   name = "dev-cluster"
+   domain = "dev-cluster.local"
+   mode = "nat"
+   addresses = ["10.0.200.0/24"]
+}
+
+resource "libvirt_volume" "debian9-base" {
+  name = "debian9-base"
+  source = "http://localhost:8000/debian9-uefi.qcow2"
+}
+
+resource "libvirt_volume" "volume" {
+  name = "vm${count.index}"
+  base_volume_id = "${libvirt_volume.debian9-base.id}"
+  count = 2
+}
+
+resource "libvirt_domain" "domain" {
+  name = "vm${count.index}"
+  memory = "256"
+  vcpu = 1
+  nvram = "/var/lib/libvirt/qemu/nvram/vm${count.index}_VARS.fd"
+  nvram_template = "/srv/provisioning/terraform/debian-stable-uefi_VARS.fd"
+  firmware = "/usr/share/OVMF/OVMF_CODE.fd"
+  network_interface {
+    network_name = "dev-cluster"
+  }
+  disk {
+    volume_id = "${element(libvirt_volume.volume.*.id, count.index)}"
+  }
+  graphics {
+    type = "spice"
+    listen_type = "address"
+    autoport = true
+  }
+  count = 2
```

Comment:
No problems in doing that. I found it a bit weird to have it down there, but then again that's how it was done on other examples, so I imagine it was a convention or something...